.travis.yml

```diff
@@ -1,17 +1,37 @@
-language: rust
-rust:
-  - stable
-  - beta
-  - nightly
+dist: trusty
 sudo: false
+language: rust
+
+matrix:
+  fast_finish: true
+  include:
+    - name: "stable"
+      rust: stable
+    - name: "beta"
+      rust: beta
+    - name: "nightly"
+      rust: nightly
+      after_success:
+        - travis-cargo --only nightly doc-upload
+    - name: "libc-test"
+      rust: nightly
+      os: osx
+      osx_image: xcode9.4
+      after_script:
+        - git clone https://github.com/rust-lang/libc
+        - sed -i '' 's@ctest = "0.2.2"@ctest = { path = "../.." }@g' libc/libc-test/Cargo.toml
```

Comment:
How come this is only done on the OSX target?

Reply:
Because the `libc-test` work on OSX "as is". For Linux and other targets I'll have to include the `libc` docker containers etc. so that the versions of the header files match. Could be done, but then we'll have to keep changes to the ci in sync with `libc`.

Reply:
@alexcrichton could you activate a daily cron-job for `ctest` ? This will break once we update `libc` to `ctest `0.2.3`, but I'll send a PR when that happens to fix that.